_includes/footer.html

```diff
@@ -45,7 +45,7 @@ <h6 class="Footer__list-head">More</h6>
           <a href="https://tools.debtcollective.org/contact">Contact</a>
         </div>
         <div class="Footer__list-item">
-          <a class=".js-donate" href="#donate">Donate</a>
+          <a href="https://tools.debtcollective.org/?donate">Donate</a>
```

Comment:
Let's move this to the config file. https://github.com/debtcollective/landingpage/blob/master/_data/urls.yml

Reply:
:+1: whoops, of course.

Reply:
done, I fixed up some of the other links in the footer too